patches/react-native-keychain+6.1.1.patch

```diff
@@ -130,3 +130,29 @@ index b1a11f2..6a055c2 100644
  /**
   * Deletes all internet password keychain entries for `server`.
   * @param {string} server URL to server.
+diff --git a/node_modules/react-native-keychain/typings/react-native-keychain.d.ts b/node_modules/react-native-keychain/typings/react-native-keychain.d.ts
+index e813ef0..5253282 100644
+--- a/node_modules/react-native-keychain/typings/react-native-keychain.d.ts
++++ b/node_modules/react-native-keychain/typings/react-native-keychain.d.ts
+@@ -9,6 +9,10 @@ declare module 'react-native-keychain' {
+     password: string;
+   }
+ 
++  export interface AllCredentials {
++    results: UserCredentials[];
++  }
++
+   export interface SharedWebCredentials extends UserCredentials {
+     server: string;
+   }
+@@ -109,6 +113,10 @@ declare module 'react-native-keychain' {
+     options?: Options
+   ): Promise<false | UserCredentials>;
+ 
++  function getAllInternetCredentials(): Promise<false | AllCredentials>;
++
++  function getAllInternetCredentialsKeys(): Promise<false | AllCredentials>;
```

Comment:
Does getAllInternetCredentialsKeys technically return <false | string[]> ?

Reply:
No, it's AllCredentials too, the only difference is that it doesn't include the password or server, but it's still an object